src/towncrier/__init__.py

```diff
@@ -103,8 +103,7 @@ def __main(draft, directory, project_version, project_date, answer_yes):
             "What is seen below is what would be written.\n",
             err=to_err,
         )
-        click.echo(top_line, err=to_err)
-        click.echo(rendered)
+        click.echo("%s%s" % (top_line, rendered))
```

Comment:
Looks like this breaks the tests since you dropped the newline.
Maybe just remove the `err` kwarg?